client/proto/generate.sh

```
@@ -1,17 +1,16 @@
-then
-  echo realpath is not installed
-  echo run: brew install coreutils
-  exit 1
+if ! which realpath >/dev/null 2>&1; then
+	echo realpath is not installed
+	echo run: brew install coreutils
+	exit 1
 fi
 
 old_pwd=$(pwd)
 script_path=$(dirname $(realpath "$0"))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)